.swiftlint.yml

```diff
@@ -8,8 +8,8 @@ line_length:
     ignores_comments: true
     ignores_urls: true
 function_body_length:
-    warning: 40
-    error: 60
+    warning: 60
+    error: 100
```

Comment:
I'd prefer you disable it locally

Reply:
will the ci not check for the same?

Reply:
Oh sorry I'm not clear enough, disable lint body length check on your method:
```// swiftlint:disable function_body_length```

Reply:
fixed it thanks for the reminder!